src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigSearch">
+      <div class="icon1">
+        <img src="images/Search.svg" alt="searchIcon">
+      </div>
+      <form
+        data-qa="big"
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post">
+        <input
+          type="text"
+          data-qa="keypress"
+          class="searchbar1"
+          placeholder="Try &quot Los Angeles &quot">
+      </form>
+    </div>
+
+    <div class="smallSearch">
+      <div >
```

Comment:
redundant div
also make structure of form simplier, without div
icon you can make as pseudoelement for label (better solution)
```
<form>
  <label>
    <input />
  </label>
</form>